freqtrade/strategy/interface.py

```diff
@@ -174,6 +174,7 @@ def stop_loss_reached(self, current_rate: float, trade: Trade, current_time: dat
         """
         Based on current profit of the trade and configured (trailing) stoploss,
         decides to sell or not
+        :param current_profit: current profit in percent
```

Comment:
i had to add this as i stumbed for half an hour until i realized this is percent, not absolute!